dotnet/flatpak-dotnet-generator.py

```diff
@@ -45,6 +45,9 @@ def main() -> None:
         help="The directory the generated sources file will save sources to",
         default="nuget-sources",
     )
+    parser.add_argument(
+        "--only-arches", help="limit the outout to this flatpak arch", default=None
```

Comment:
```suggestion
        "--only-arches", help="Limit the source to this Flatpak arch", default=None
```

Reply:
I believe there is a list, if unwritten. Heres a document outlining the concept. https://learn.microsoft.com/en-us/dotnet/core/rid-catalog

The list is likely pretty long since the identifers seem to essentially be combinations of OSses and architectures, i.e. `linux-x64`, `unix-x64`, as well as just OSses (`linux`). Same goes for macos and windows too.

Should I only limit it to the linux ones? maybe including Unix too? or should this include macos and windows too?